
errno.h needs to be included explicitly ...

  - Rene Rebe <rene@exactcode.de>

--- ./src/Screen.cc.vanilla	2004-11-06 17:13:12.615169843 +0100
+++ ./src/Screen.cc	2004-11-06 17:13:40.718495227 +0100
@@ -59,6 +59,8 @@
 #endif // HAVE_SIGNAL_H
 }
 
+#include <errno.h>
+
 #include <algorithm>
 #include <sstream>
 #include <iostream>
